bin/queue

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Application;
+use Yiisoft\Di\Container;
+use Yiisoft\Di\ContainerConfig;
+
+$autoload = $GLOBALS['_composer_autoload_path'] ?? './vendor/autoload.php';
```

Comment:
```suggestion
$autoload = $_composer_autoload_path ?? dirname(__DIR__, 3) . '/vendor/autoload.php';
```

https://getcomposer.org/doc/articles/vendor-binaries.md#finding-the-composer-autoloader-from-a-binary